backend/s3/s3.go

```diff
@@ -1180,12 +1180,18 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Initialize pacer to 0 retries because we are relying on SDK with
+	// MaxRetries for that
```

Comment:
Dot at the end maybe.

Reply:
Consistent with rclone comments where there is no dot.